tsconfig.json

```diff
@@ -25,7 +25,7 @@
 
     /* Modules */
     "module": "es2022" /* Specify what module code is generated. */,
-    // "rootDir": "./",                                  /* Specify the root folder within your source files. */
+    "rootDir": "./",                                  /* Specify the root folder within your source files. */
```

Comment:
this was the fixing change, could be that it would have worked with the old dependencies anyway